stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Attachment.kt

```diff
@@ -40,9 +49,8 @@ public data class Attachment(
     @IgnoreDeserialisation
     var uploadState: UploadState? = null,
 
-    @IgnoreSerialisation
-    @IgnoreDeserialisation
```

Comment:
Are we sure about this change? We were ignoring this so that Gson doesn't try to read a map called `extraData` (because it's never there), because we're populating and serializing this into the top level object manually.

Reply:
I talked with @zsmb13 and the this gets available anyways, we don't need to serialise it

Reply:
Also tested, and the behaviour it still the same